src/components/ActNowCard/PetitionCard.js

```diff
@@ -18,11 +20,17 @@ function PetitionCard({ closeCard, promiseActNow, ...props }) {
 
   const { petitionJoin, petitionTitle: petitionStart } = props;
   const [open, setOpen] = useState(false);
+  const { session } = useSession();
 
   const classes = useStyles();
 
   const handleFormOpen = () => {
-    setOpen(true);
+    if (!session) {
+      setOpen(false);
```

Comment:
Form should be closed at this point, no?

Reply:
@kilemensi Correct 🚀